app/controllers/api/v1/short_form_controller.rb

```diff
@@ -38,6 +35,12 @@ def delete_proof
 
   def submit_application
     response = ShortFormService.create(application_params)
+    if application_params[:primaryApplicant][:email].present?
```

Comment:
might be cool to write a test for this @kaptron or capture as a chore since I know we're at a crunch.